api/test/docker/docker-compose.yaml

```diff
@@ -149,14 +150,15 @@ services:
         ipv4_address: 172.16.238.30
 
   apisix2:
+    hostname: apisix_server2
     build:
       context: ../../
       dockerfile: test/docker/Dockerfile-apisix
       args:
         - APISIX_VERSION=2.1
```

Comment:
we should use `master` as APISIX_VERSION here.
`server-info` is new plugin that not in APISIX 2.1.

Reply:
OK